react-validated-input/my-app/src/ValidatedInput.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { FcCheckmark } from 'react-icons/fc';
+import { FaTimes } from 'react-icons/fa';
+
+export function ValidatedInput() {
+  const [charCount, setCharCount] = useState(0);
+  const label = <label htmlFor="password">Password:</label>;
```

Comment:
I think component code is easier to read if you put this all down in the `return` statement. Then it looks more like HTML and it's easier to see the structure of the HTML.